.travis/build-types-v2.ts

```diff
@@ -1,7 +1,7 @@
 import * as fs from 'fs';
```

Comment:
Move this to `scripts` folder or smth since it isn't related to travisci anymore